interfaces/policy/helpers.go

```diff
@@ -66,6 +67,19 @@ func checkID(kind, id string, ids []string, special map[string]string) error {
 	return fmt.Errorf("%s does not match", kind)
 }
 
+func checkOnClassic(c *asserts.OnClassicConstraint) error {
+	if c == nil {
+		return nil
+	}
+	if c.Classic != release.OnClassic {
+		return fmt.Errorf("on-classic mismatch")
```

Comment:
Not sure if people will get it anyway, but perhaps:

"on a classic operating system" and
"not on a classic operating system"